csharp/ql/lib/semmlecode.csharp.dbscheme.stats

```diff
@@ -12,6 +12,10 @@
             <k>@extractor_message</k>
             <v>41551</v>
         </e>
+        <e>
+            <k>@externalDataElement</k>
```

Comment:
As far as I know, `codeql test` will throw an exception if you have a `typesizes` with a `0` entry. I _think_ the motivation for the now-reverted PR is that the stats generated by DCA don't include stats from tests, so we cannot provide a non-zero `typesizes` entry for `externalDataElement`.

What we're doing in C++ to address this issue is to add an entry in the `static-stats/cpp` folder in the DCA repo. I imagine C# could do the same if it's needed.

Reply:
There are several other, existing `typesizes` entries with a `0` value. I'm a bit confused about why they don't cause a problem.

Reply:
Good point. I guess this shouldn't block the revert. But I'd definitely be interested in knowing why `codeql test` doesn't complain too 🤔.